tests/async_mock.py

```diff
@@ -1,9 +1,3 @@
 """Mock utilities that are async aware."""
-import sys
 
-if sys.version_info[:2] < (3, 8):
-    from asynctest.mock import *  # noqa
-
-    AsyncMock = CoroutineMock  # noqa: F405
-else:
-    from unittest.mock import *  # noqa
+from unittest.mock import *  # noqa
```

Comment:
Can we remove the `asynctest` requirement completely?

Reply:
Yes, but that is a bigger refactor for a separate PR I would say...

Reply:
Created PR to drop asynctest: <https://github.com/home-assistant/core/pull/44746>